e2e/support/cypress_sample_instance_data.js

```diff
@@ -0,0 +1,31 @@
+/**
+ *  This JSON file gets recreated every time Cypress starts.
+ *  See: `e2e/snapshot-creators/default.cy.snap.js:19`
+ *
+ *  - It had to be added to `.gitignore`.
+ *  - It contains extracted metadata from the default instance state (like question and dashboard ids)
+ */
+
+import _ from "underscore";
+
+// eslint-disable-next-line import/no-unresolved
+import SAMPLE_INSTANCE_DATA from "./cypress_sample_instance_data.json";
```

Comment:
It would be helpful to maybe export this whole thing to provide access to all data. We can do this later (or not). Just thinking out loud.